src/PrestaShopBundle/Form/Admin/Product/ProductOptions.php

```diff
@@ -204,7 +204,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder->add('attachment_product', 'PrestaShopBundle\Form\Admin\Product\ProductAttachement', array(
             'required' => false,
             'label' => $this->translator->trans('Attachment', [], 'Admin.Catalog.Feature'),
-            'attr' => ['data-action' => $this->router->generate('admin_product_attachement_add_action')]
+            'attr' => ['data-action' => $this->router->generate('admin_product_attachement_add_action', array('idProduct' => 1))]
```

Comment:
Why not, if you use it [there](https://github.com/PrestaShop/PrestaShop/blob/1.7.0.0/src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig#L394), [there](https://github.com/PrestaShop/PrestaShop/blob/1.7.0.0/src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig#L639) and [there](https://github.com/PrestaShop/PrestaShop/blob/1.7.0.0/src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig#L911) ? :)